<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>132</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>240</width>
    <height>132</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>240</width>
    <height>132</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Terminar Tanda</string>
  </property>
  <widget class="QComboBox" name="comboBoxTandas">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCerrar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>88</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Terminar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancelar">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>88</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
